maxim/tests/test_litellm.py

```
@@ -217,6 +239,9 @@ async def test_openai_async(self) -> None:
         # Remove the long sleep for faster testing
         # await asyncio.sleep(10)
 
+        # Give LiteLLM callback time to complete
+        time.sleep(0.5)
```

Ruff (0.11.9)

Async functions should not call `time.sleep`

(ASYNC251)